SQL/migrate/V053_preference_asfx_instrument.sql

```diff
@@ -0,0 +1,6 @@
+--
+-- Set default 'On' for 'asfx_instruments'
+-- 
+
+ALTER TABLE `ss13_characters`
+	ALTER 'asfx_togs' SET DEFAULT '128';
```

Comment:
This wasn't exactly what I had in mind. Or rather, this doesn't update the existing entries.

```suggestion
UPDATE `ss13_characters` SET `asfx_togs` = `asfx_togs` | 128;
```

Reply:
Also, the file name needs two underscores between the version number and the rest. so `V053__preference_asfx_intr...`